molo/core/models.py

```diff
@@ -45,6 +79,29 @@ class Main(Page):
     parent_page_types = []
     subpage_types = ['core.LanguagePage']
 
+    COMMENTING_STATE_CHOICES = (
+        ('O', 'Open'),
+        ('C', 'Closed'),
+        ('D', 'Disabled'),
+        ('T', 'Timestamped'),
+    )
+    commenting_state = models.CharField(max_length=1,
+                                        choices=COMMENTING_STATE_CHOICES,
+                                        blank=False,
```

Comment:
Why is this one allowing `blank=False` whereas the other two are `blank=True`? Is it because this one has a default?